understanding/20/audio-control.html

```diff
@@ -11,12 +11,12 @@ <h1>Understanding Audio Control</h1>
    <section id="brief">
 		<h2>In brief</h2>
 		<dl>
-		<dt>Objective</dt><dd>A page that plays music or sounds doesn't disrupt people</dd>
-		<dt>Author task</dt><dd>If you play audio content automatically, let people turn it down or off</dd>
-		
-		</dl>
-		
-		</section>
+         <dt>Goal</dt><dd>A page that plays music or sounds doesn't disrupt people.</dd>
+         <dt>What to do</dt><dd>If you play audio content automatically, let people turn it down or off.</dd>
+         <dt>Why it's important</dt><dd>Sound can distract some users or interfere with their assistive technology.</dd> 
```

Comment:
Should this "Why it's important" match the form of others? Most seem to be expressed as positive impacts for people with disabilities. Perhaps something like "Sound does not distract users or interfere with their assistive technology."